net/bully/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bully
+PKG_VERSION:=1.1
+PKG_RELEASE:=31
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/aanarchyy/bully/archive/$(PKG_VERSION).tar.gz
```

Comment:
URL shouldn't contain file name

Reply:
I prefer that the latest revision is used instead of version 1.1. It's 2016 vs. 2017.

Reply:
Then `PKG_SOURCE` should be `$(PKG_VERSION).tar.gz`
And `PKG_SOURCE URL` should be `https://github.com/aanarchyy/bully/archive/`

If not, wget wont find the file in github.

Reply:
Ok, so then let's use "master"

Reply:
```$(PKG_VERSION).tar.gz``` is a no go, either use codeload (grep the tree) or git over https.

Reply:
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/aanarchyy/bully/tar.gz/$(PKG_VERSION)?